### PR TITLE
Add ingest API get methods for media objects

### PR DESCRIPTION
**Pull request by bkeese**

### CODE REVIEW

File: app/controllers/media_objects_controller.rb [hunk 1]
```diff
@@ -95,8 +95,13 @@ def custom_update
     flash[:notice] = @notice
   end
 
+  def index
+    respond_to do |format|
+      format.json   { render json: MediaObject.all.map{|mo| mo.to_json } }
```
Comment (cjcolvar) [inline app/controllers/media_objects_controller.rb]: I think we need to introduce pagination here and maybe back on collection#index as well.

File: app/models/media_object.rb [hunk 2]
```diff
@@ -417,8 +417,14 @@ def to_solr(solr_doc = Hash.new, opts = {})
...
     {
-      id: self.pid,
-      title: self.title
+      id: pid,
+      title: title,
+      collection: collection.name,
+      main_contributors: creator, 
+      publication_date: date_created, 
+      published_by: avalon_publisher,
+      published: !avalon_publisher.nil?,
```
Comment (cjcolvar) [inline app/models/media_object.rb]: why not use #published? instead?

File: app/controllers/media_objects_controller.rb (hunk 1 above)
Comment (bkeese) [inline app/controllers/media_objects_controller.rb]: I kind of like this approach: http://metabates.com/2012/02/22/adding-pagination-to-an-api/ Or is there another way you had in mind?

File: app/models/media_object.rb (hunk 2 above)
Comment (bkeese) [inline app/models/media_object.rb]: Good point. Now why didn't I think of that?

File: app/controllers/media_objects_controller.rb (hunk 1 above)
Comment (cjcolvar) [inline app/controllers/media_objects_controller.rb]: I'm looking into this right now: http://jsonapi.org/format/#fetching-pagination

**Comment by bkeese** [comment]
Rebased from develop to pull in ActiveFedora 8.1.0

**Pull request merged**